src/s2geography/geoarrow_test.cc

```diff
@@ -202,3 +205,61 @@ TEST(GeoArrow, GeoArrowReaderReadWKTCollection) {
   EXPECT_EQ(linestring->edge(0).v0, S2LatLng::FromDegrees(1, 0).ToPoint());
   EXPECT_EQ(linestring->edge(0).v1, S2LatLng::FromDegrees(3, 2).ToPoint());
 }
+
+using s2geography::geoarrow::Writer;
+
+TEST(GeoArrow, GeoArrowWriterPoint) {
+  s2geography::WKTReader reader;
+  auto geog1 = reader.read_feature("POINT (0 1)");
+  auto geog2 = reader.read_feature("POINT (2 3)");
+
+  Writer writer;
+  nanoarrow::UniqueSchema schema;
+  nanoarrow::UniqueArray array;
+
+  InitSchemaGeoArrowPoint(schema.get());
+  writer.Init(schema.get());
+
+  writer.WriteGeography(*geog1);
+  writer.WriteGeography(*geog2);
+  writer.Finish(array.get());
+
+  EXPECT_EQ(array->length, 2);
+
+  // EXPECT_THAT(nanoarrow::ViewArrayAs<double_t>(array->children[0]), ElementsAre(DoubleEq(0.0), DoubleEq(2.0)));
+  auto xs = reinterpret_cast<const double*>(array->children[0]->buffers[1]);
+  EXPECT_DOUBLE_EQ(xs[0], 0.0);
+  EXPECT_DOUBLE_EQ(xs[1], 2.0);
+  // EXPECT_THAT(nanoarrow::ViewArrayAs<double_t>(array->children[1]), ElementsAre(DoubleEq(1.0), DoubleEq(3.0)));
+  auto ys = reinterpret_cast<const double*>(array->children[1]->buffers[1]);
+  EXPECT_DOUBLE_EQ(ys[0], 1.0);
+  EXPECT_DOUBLE_EQ(ys[1], 3.0);
+}
+
+TEST(GeoArrow, GeoArrowWriterPointProjected) {
+  s2geography::WKTReader reader;
+  auto geog1 = reader.read_feature("POINT (0 1)");
+  auto geog2 = reader.read_feature("POINT (2 3)");
+
+  Writer writer;
+  nanoarrow::UniqueSchema schema;
+  nanoarrow::UniqueArray array;
+
+  s2geography::geoarrow::ExportOptions options;
+  options.set_projection(s2geography::pseudo_mercator());
```

Comment:
Are we OK with those projections being in the top-level namespace, actually?

(or if we do so, should there be something about "projection" in the name of those constructor functions?)